src/renderHook.tsx

```diff
@@ -8,16 +8,12 @@ export type RenderHookResult<Result, Props> = {
   unmount: () => void;
```

Comment:
[nit] Comparing to RTL typing, their `rerender` `Props` argument is optional: https://github.com/testing-library/react-testing-library/blob/6b4180e71286cef86a359435697965e59d408d91/types/index.d.ts#L105

Not sure if we should follow that as well.

Reply:
I don't really understand how it work on their side but what happens when we call rerender without props is that it will render the component with undefined as props and I think the expected outcome would be to rerender with the previous props. We could implement it but I'm not sure I see a use case for it